### DISCUSSION TITLE
Is there a way to skip the animation  and see the end layout - with @visx/network and d3-force

### DISCUSSION THREAD

**Discussion by hyfydistro** [airbnb/visx#1320] [Q&A]
Is there a way to skip the animation  and see the end layout - with @visx/network and d3-force

Often I see this line of code  when building the network graph :` forceRef.current.on("tick", () => forceUpdate());`

Is there a way to just see the end result of the network graph? No animation, just the end display.

**Comment by williaster** [comment]
Hi @hyfydistro , do you have a code sandbox you can share? Network force layouts run asynchronously, so it's not "just an animation" it is **computing the layout**. That means you can either 
1) show the animation, or 
2) show nothing until the animation is complete (may take seconds) and then show the final layout
